cmd/crictl/container.go

```diff
@@ -590,8 +590,8 @@ func ListContainers(client pb.RuntimeServiceClient, opts listOptions) error {
 	if err != nil {
 		return err
 	}
+	sort.Sort(containerByCreated(r.GetContainers()))
```

Comment:
I think getContainersList should include sort.Sort inside.

Reply:
It is more completer in logic.

Reply:
Agree. Will do.

Reply:
Done.